src/misc/unicode.cpp

```diff
@@ -496,6 +496,98 @@ bool Grapheme::operator<(const Grapheme& other) const
 	return false;
 }
 
+// ***************************************************************************
+// Helpers for control codes handing
+// ***************************************************************************
+
+// Constants to detect whether DOS character is a control code
+constexpr uint8_t control_code_delete    = 0x7f;
+constexpr uint8_t control_code_threshold = 0x20;
```

Comment:
UpperCamelConstants

Applied to all `constexpr` in this file.

Reply:
OK, changed.